syntax/fetlang.vim

```diff
@@ -86,6 +81,11 @@ highlight link fetlangSpecial Special
 
 " }}}
 
+highlight Comment ctermfg=LightGreen cterm=none
+highlight Keyword ctermfg=DarkBlue cterm=bold
+highlight Function ctermfg=cyan term=none
+highlight Special cterm=bold
```

Comment:
I merged but I think we should remove color definitions. This is not a color scheme and it should not define any its colors. If you want `Keyword`s are blue, that is almost belongs to your `~/.vim/color/` directory.

Reply:
I've never used vimscript so I don't really know anything, but before I added those, several of the categories were showing up as the same color. Is there another fix?

Reply:
You can `:help group-name` to look up syntax group names. If they are showing up the same color, that's about your color scheme. You should change your color scheme or modify it. Some color schemes prefer to show some syntax groups with same color, that's their problem.

Reply:
Gotcha, thanks

Reply:
By the way, vim has bunch of built-in color schemes, you can write `:colorscheme` and hit <kbd>Tab</kbd>, vim will auto complete available ones. _You may need to `:set wildmenu` for auto completion.